src/simulation/m_time_steppers.fpp

```diff
@@ -1041,6 +1246,17 @@ contains
 
         @:DEALLOCATE(rhs_vf)
 
+        ! Deallocating the cell-average RHS variable for adaptive method, Lagrangian solver
+        if (lag_bubbles) then
+            do i = 1, 6
```

Comment:
where does this 6 come from? it appears to be a magic number

Reply:
6 is the number of time stages needed for the rkck stepper. All the needed constants were placed in m_constants.